src/TTextEdit.cpp

```diff
@@ -993,13 +994,49 @@ void TTextEdit::mouseMoveEvent( QMouseEvent * event )
         }
     }
     x = ( event->x() / mFontWidth ) - timeOffset;
-    if( ( x < 0 ) || ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
+    if( ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
```

Comment:
Well if `x` ___or___ `y` was less than zero then this method would abort.  Now if x is less than zero __it does not__ so going forward everywhere that uses `x` will have to handle it (by including the timestamp but also treating x as if it WAS 0)? :thinking: